kiwi/system/prepare.py

```diff
@@ -94,18 +98,21 @@ def __init__(
         # in order to delay the Uri destructors until the System instance
         # dies. This is needed to keep bind mounted Uri locations alive
         # for System operations
-        self.uri_list = []
+        self.uri_list: List[Uri] = []
 
-    def setup_repositories(self, clear_cache=False, signing_keys=None):
+    def setup_repositories(
+        self, clear_cache: bool = False, signing_keys: List = None
```

Comment:
Would it be worth to include the type of the list elements? so instead of `signing_keys: List = None` -> `signing_keys: List[str] = None`. AFAIK this is a valid syntax, isn't it. I am also fine if we agree not to reach that level of detail, but since I saw the  `List[Uri]` few lines above I am wondering about the criteria around this topic.

Reply:
+1 on this, before noticing prepare was taken, I was following that approach.

Reply:
I agree, I was just not sure if this is always a list of str ? can we be sure here ? In the other case I knew it for sure

Reply:
ok this ends in an import of a key which is a file path. Thus List[str] is good. I'll adapt

Reply:
done